File: Server (Java Spring Boot)/Hack_with_CW/src/main/java/com/server/attendance/dto/ResponseDTO.java
package com.server.attendance.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import org.springframework.http.HttpStatus;

@Getter
@Setter
@AllArgsConstructor
public class ResponseDTO {

    int status;
    HttpStatus httpStatus;
    String message;

}

File: Fontend (Android App)/College_Attendance_Student/lib/common/models/response_dto.dart
class ResponseDTO {
  final int status;
  final String httpStatus;
  final String message;

  const ResponseDTO({
    this.status,
    this.httpStatus,
    this.message,
  });

  factory ResponseDTO.fromJson(Map<String, dynamic> json) {
    return ResponseDTO(
      status: json['status'],
      httpStatus: json['httpStatus'] ?? "",
      message: json['message'] ?? "",
    );
  }
}
